frontend/package.json

```diff
@@ -69,6 +71,8 @@
     "eslint-config-prettier": "6.7.0",
     "eslint-plugin-prettier": "3.1.1",
     "husky": "4.2.3",
+    "jest-environment-jsdom-fourteen": "^1.0.1",
```

Comment:
Could you please comment why we need to include this? I checked the version of `react-scripts` we are using and it bundles this dependency so it would seem we shouldn't need to include it here as well. Unless there's something else?

Reply:
testing with ag-grid requires this library to run

Reply:
As stated [here](https://github.com/facebook/create-react-app/issues/7491#issuecomment-520157683) all you would need to do to enable `jest-environment-jsdom-fourteen` is:

```
//package.json
"test": "react-scripts test --env=jsdom-fourteen"
```

I see you did that below. Could you please try removing the dependency from our **package.json** and see if it works with the test? If possible I'd try to avoid bringing extra dependencies. Thanks!

Reply:
You, sir, are right, the dependency is not needed